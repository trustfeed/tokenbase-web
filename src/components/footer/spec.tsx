import * as React from 'react';
import Footer from '.';
import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
