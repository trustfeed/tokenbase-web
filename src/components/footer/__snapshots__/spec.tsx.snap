// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="footer text-center"
>
  <span
    className="text-center copyright"
  >
    Copyright Â© TrustFeed 2018
  </span>
</footer>
`;
